scripts/llnl/common_build_functions.py

```diff
@@ -160,7 +159,7 @@ def uberenv_build(prefix, spec, project_file, mirror_path):
     Calls uberenv to install tpls for a given spec to given prefix.
     """
     assertUberenvExists()
-    cmd  = "python {0} -k ".format(get_uberenv_path())
+    cmd  = "{0} {1} -k ".format(sys.executable, get_uberenv_path())
```

Comment:
You are just running into all the problems this PR arent you....

Reply:
To be fair this one was (mostly) my fault since there's some Python2-incompatible logic I added to uberenv.